src/components/CommentList/CommentList.jsx

```diff
@@ -1 +1,13 @@
-export const CommentList = () => <>Put the list here</>;
+import React from 'react';
+import { CommentInfo } from '../CommentInfo';
+import './CommentList.scss';
+
+export const CommentList = ({ comments }) => (
+  <ul className="CommentList">
+    {comments.map(comment => (
+      <li key={comment.id} className="CommentList__item">
```

Comment:
I think the `li` tag itself can be placed inside the `CommentInfo` component as well

Reply:
> I think the `li` tag itself can be placed inside the `CommentInfo` component as well

Good point